tstest/resource.go

```diff
@@ -32,6 +33,11 @@ func NewResourceCheck() *ResourceCheck {
 	return r
 }
 
+// Quiet prevents r from logging except on failure.
+func (r *ResourceCheck) Quiet() {
```

Comment:
Can we make it quiet always? Does it need a toggle?

Reply:
I couldn't figure out from the git history why it is this way. I'll make it always quiet instead.